clab/ixia-c-b2b-lag/otg.yaml

```diff
@@ -91,11 +91,11 @@ layer1:
   promiscuous: true
   speed: speed_1_gbps
 ports:
-- location: te1:5555
+- location: te1:5555;1
```

Comment:
Can we also add values to dest port to get the traffic to actually hash between tx ports.

Reply:
Done